docs/de/README.md

```diff
@@ -57,7 +57,8 @@ In der YAML-Datei können die folgenden Keywörter definiert werden:
 |-|-| 
 | __title__       | Der Titel der Extension| 
 | __description__ | Lange Beschreibung der Extension, die in der Detailansicht erscheint | 
-| __keywords__    | Liste von Keywörtern für die Suche | 
+| __keywords__    | Liste von Keywörtern für die Suche |
+| __homepage__    | (optional) Wird vom Linter gefordert, wenn es sich um ein privates Paket handelt |
```

Comment:
How about something like "Nur erforderlich wenn es sich um ein privates Paket handelt". I think the optional + required combination would be confusing.

Reply:
Which would you prefer - which would be better? - "only required"?
yesterday we had an error due to an undocumented key 'homepage'. The key is required, but only if it is a private package, otherwise it is optional. we thought it could be documented with.
https://github.com/contao/package-metadata/blob/6cefde62e763a477022199c027b62377ea7cfc73/linter/src/LintCommand.php#L151

Reply:
Yes that would bee good to document! It is only required and useful for a private package, because public packages have a page on packagist.org and can also have a homepage key in the composer.json